app/main.py

```diff
@@ -1,3 +1,76 @@
-def shop_trip():
-    # write your code here
-    pass
+import datetime
+import json
+from app.customer import Customer
+from app.shop import Shop
+
+
+def calculate_product_cost(product_cart: dict, products: dict) -> float:
+    return sum(product_cart[product] * products[product]
+               for product in product_cart)
+
+
+def format_price(price: float) -> str:
+    if price % 1 == 0:
+        return str(int(price))
+    else:
+        return f"{price}"
```

Comment:
I think you can remove this method and use `round(..., 2)` only . 
Also second return is redundat here